src/logger.ts

```diff
@@ -1,16 +1,15 @@
 import winston from 'winston';
-import minimist from 'minimist';
+import { logLevel, logLocation } from './config';
 
 const logger = winston.createLogger({
-    level: 'info',
+    level: logLevel || 'info',
     format: winston.format.combine(
         winston.format.timestamp(),
         winston.format.json()
     ),
     transports: [
         new winston.transports.File({
-            filename:
-                minimist(process.argv.slice(2)).logLocation || 'combined.log',
+            filename: logLocation || 'combined.log',
```

Comment:
Maybe not for this PR but I think the default should be stdout.

Reply:
I changed it to be so, the combined.log default was strange.